src/pages/index.js

```diff
@@ -15,16 +15,83 @@ import {
   cardList,
   templateSelector,
   options,
+  profileTitle,
+  profileImage,
+  profileDescription,
+  updateAvatarContainer,
+  updateAvatarForm,
 } from "../utils/constants.js";
 import "./index.css";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 
+//////////////////////
+// ----- APIs ----- //
+//////////////////////
+
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/group-12",
+  headers: {
+    authorization: "02a3c077-82c6-4270-8df6-e8c1f2fec9e8",
+    "content-type": "application/json",
+  },
+});
+
+api
+  .getUserInformation()
```

Comment:
You can receive all the initial data together using `Promise.all`, so you will save user's id and provide it to Card with constructor arguments and will be able to define if user is card's owner

```js
let userId;

Promise.all([api.getUserInfo(), api.getInitialCards()]).then(([user, cards]) => {
  userInfo.setProfileInfo(userData.name, userData.about);
  userId = user._id;  // set user id
  // initialize cards section
}).catch(() => console.log(...))
```